Editor/Editors/CinemachineVirtualCameraBaseEditor.cs

```diff
@@ -356,7 +356,7 @@ protected void DrawGlobalControlsInInspector()
                     MessageType.Info);
         }
 
-        LensSettingsInspectorHelper m_LensSettingsInspectorHelper;
+        private protected LensSettingsInspectorHelper m_LensSettingsInspectorHelper;
```

Comment:
I think I'd prefer that you just expose a "DisplayHorizontalFOV" getter that looks at the private LensSettingsInspectorHelper, rather than this kooky private protected thing